packages/react-devtools-shared/src/devtools/views/Components/OwnersStack.css

```diff
@@ -73,27 +73,32 @@
   background: var(--color-button-background-focus);
 }
 
-.Modal[data-reach-menu-list] {
+.ModalDark[data-reach-menu-list], .ModalLight[data-reach-menu-list] {
```

Comment:
Move color-specific stuff into a separate rule? Will also allow us to more easily compare light vs dark.

Reply:
Yes totally makes sense 🙌